### PR TITLE
Added the error coverage for customerAttributeMetadata query

### PR DESCRIPTION
**Pull request by dineshvb**
## Purpose of this pull request

This pull request (PR) adds the error coverage for customerAttributeMetadata query.

## Affected DevDocs pages

-   /guides/v2.3/graphql/queries/custom-attribute-metadata.html
-   /guides/v2.4/graphql/queries/custom-attribute-metadata.html

### CODE REVIEW

File: src/guides/v2.3/graphql/queries/custom-attribute-metadata.md [hunk 1]
```diff
@@ -232,3 +232,14 @@ Attribute |  Data Type | Description
 --- | --- | ---
 `label` | String | The name of an attribute option
 `value` | String | The value assigned to an attribute option
+
+## Errors
+
+Error | Description
+--- | ---
+`Field "customAttributeMetadata" argument "attributes" of type "[AttributeInput!]!" is required but not provided` | The `attributes` array parameter is required.
+`The attribute with a "xxxx" attributeCode doesn't exist. Verify the attribute and try again` | The given `attribue_code` parameter is invalid.
```
Comment (BarnyShergold) [inline src/guides/v2.3/graphql/queries/custom-attribute-metadata.md:241]:
```suggestion
`The attribute with a "xxxx" attributeCode doesn't exist. Verify the attribute and try again` | The given `attribute_code` parameter is invalid.
```

File: src/guides/v2.3/graphql/queries/custom-attribute-metadata.md [hunk 2]
```diff
@@ -232,3 +232,14 @@ Attribute |  Data Type | Description
...
+`Invalid entity_type specified: "xxxx"` | The given `entity_type` is invalid.
+`Missing attribute_code for the input entity_type: "xxxx"`| There is no value mentioned for the `attribute_code` parameter for the given `entity_type` parameter.
```
Comment (BarnyShergold) [inline src/guides/v2.3/graphql/queries/custom-attribute-metadata.md:243]:
```suggestion
`Missing attribute_code for the input entity_type: "xxxx"`| There is no value passed for the `attribute_code` parameter for the given `entity_type` parameter.
```

File: src/guides/v2.3/graphql/queries/custom-attribute-metadata.md [hunk 3]
```diff
@@ -232,3 +232,14 @@ Attribute |  Data Type | Description
...
+`Missing entity_type for the input attribute_code: "xxxx"` | There is no value mentioned for the `entity_type` parameter for the given `attribute_code` parameter.
```
Comment (BarnyShergold) [inline src/guides/v2.3/graphql/queries/custom-attribute-metadata.md:244]:
```suggestion
`Missing entity_type for the input attribute_code: "xxxx"` | There is no value passed for the `entity_type` parameter for the given `attribute_code` parameter.
```

File: src/guides/v2.3/graphql/queries/custom-attribute-metadata.md [hunk 4]
```diff
@@ -232,3 +232,14 @@ Attribute |  Data Type | Description
...
+`Missing attribute_code/entity_type for the input Empty AttributeInput` | There are no values mentioned for both `attribute_code` and `entity_type` parameters.
```
Comment (BarnyShergold) [inline src/guides/v2.3/graphql/queries/custom-attribute-metadata.md:245]:
```suggestion
`Missing attribute_code/entity_type for the input Empty AttributeInput` | There are no values passed for both `attribute_code` and `entity_type` parameters.
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
Just a few grammar corrections

**Review by keharper — APPROVED** [review]

**Comment by keharper** [comment]
running tests

**Comment by keharper** [comment]
running tests

**Pull request merged**